File: Wirtualny swiat-java/Animals/Sheep.java
package Animals;

import World.Organism;
import World.Animal;
import World.GameWorld;
import World.Point;

public class Sheep extends Animal {
    public Sheep(int x, int y, GameWorld gameWorld) {
        super(4, 4, x, y, gameWorld, 'S');
    }

    @Override
    public void draw() {
        System.out.print(symbol);
    }

    @Override
    public void action() {
        super.action();
    }
}

File: Wirtualny Swiat C++/animals/Sheep.cpp
//
// Created by Dorota on 17.04.2025.
//
#include "Sheep.h"
#include <iostream>
using namespace std;

#include "../Animal.h"

Sheep::Sheep(int x, int y, GameWorld* game_world): Animal(4, 4, x, y, game_world, 'S') {
}

void Sheep::draw() {
    cout << symbol;
}

void Sheep::action(){
    Animal::action(); //dziedziczenie ruchu
}